app/models/miq_action.rb

```diff
@@ -848,6 +848,31 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_scan_all(action, rec, _inputs)
```

Comment:
I don't think I understand this action.  It wants a single container image as the input, but then it grabs the `ExtManagementSystem` for that container image and scans *all* container images for the `EMS`.

It seems like it might make more sense to have the `rec` just be the `ContainerManager` and do the scan on all images from there?

@gmcculloug maybe I'm missing something fundamental about actions?